activiti-api-process-model/src/main/java/org/activiti/api/process/model/payloads/GetSubprocessesPayload.java

```diff
@@ -0,0 +1,33 @@
+package org.activiti.api.process.model.payloads;
+
+import java.util.UUID;
+
+import org.activiti.api.model.shared.Payload;
+
+public class GetSubprocessesPayload implements Payload {
```

Comment:
As my previous comment you don't need this new payload type, just reuse `GetProcessInstancesPayload`.